apps/desktop/src/lib/branch/SeriesHeader.svelte

```diff
@@ -97,15 +97,11 @@
 	});
 
 	async function handleReloadPR() {
-		await Promise.allSettled([
-			prMonitor?.refresh(),
-			checksMonitor?.update(),
-			$hostedListingServiceStore?.refresh()
```

Comment:
I don't think `HostedListingService` needs to be refreshed since its output is only relevant to displaying a pr if `pr_number` isn't available in `virtual_branches.toml`.